### ISSUE TITLE
FixedBatchNormalization issue

### ISSUE THREAD

**Issue by rohith513** [Abhijit-2592/Keras_object_detection#9]
Hello, thank you for adding validation part. Can you tell me why you used FixedbatchNormalization instead of BatchNormalization in ResNet? 
I want to freeze earlier layers in ResNet and train the rest of the model. I am not sure how this custom batchnorm would behave in this case as I am freezing batchnorm layers too. 
During training, validation loss is high and classifier accuracy decreases. I guess this has something to do with batchnorm.
